configure

```diff
@@ -1955,14 +1955,13 @@ Optional Features:
   --enable-legacy-using-namespace
                           add "using namespace libMesh" to libMesh headers
   --enable-everything     enable all non-conflicting options
-  --enable-unique-id      build with unique id suppport
+  --enable-unique-id      build with unique id support
```

Comment:
Ha!  We've been copying and pasting that triple p around for a decade, haven't we?